src/lib/components/Post.svelte

```diff
@@ -139,8 +140,15 @@
     <div class="flex h-full w-2/5 grow flex-col justify-between">
         <Card.Header class="p-2 px-6">
             <Card.Title class="space-y-1">
-                <div class="mb-4">
-                    <UserChannel poster={post ? post.poster : undefined} />
+                <div class="flex justify-between">
+                    <div class="mb-4 grow">
+                        <UserChannel poster={post ? post.poster : undefined} />
```

Comment:
This is starting to wrap with longer names. Perhaps we need to adjust some sizing here.

Reply:
Agreed.  The sizing has been adjusted to be 
![image](https://github.com/user-attachments/assets/c901da2b-348d-49b6-b79a-ca9e1ed05117)